services/minespace-web/src/constants/routes.ts

```diff
@@ -41,42 +41,49 @@ const ProjectSubmissionStatusPage = React.lazy(() =>
 export const HOME = {
   route: "/",
   component: LandingPage,
+  helpKey: "Home",
```

Comment:
Just thought I'd mention:
- currently, this is what the title of the guide is based on
- some of these pages are very difficult to access on prod without actually creating data
- I anticipate that "title" is going to be an editable field in the future to deal with the first problem
- as for accessing the pages, I anticipate that CORE users will be able to view a list of keys and pick one without actually going through the motions to get there
- I duplicated keys where I felt it was appropriate

I didn't worry about it too much about making them perfect right now, I figure that'll come out in testing.